sensor/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/camera_calibrator.py

```diff
@@ -0,0 +1,1101 @@
+                    self.evaluation_sample_slider.setRange(
+                        0, self.data_collector.get_num_evaluation_samples() - 1
+                    )
+                    self.evaluation_sample_slider.setValue(0)
+                    self.evaluation_sample_slider.setEnabled(True)
+                    self.evaluation_sample_slider.valueChanged.emit(0)
+                else:
+                    self.training_sample_slider.setEnabled(False)
+                    self.evaluation_sample_slider.setEnabled(False)
+
+            if self.pending_detection_result:
+                QTimer.singleShot(1000, delayed_change)
+            else:
+                delayed_change()
+
+        def on_training_sample_changed(index):
+            print(f"on_training_sample_changed={index}")
+            self.training_sample_label.setText(f"Training sample: {index}")
+            img = self.data_collector.get_training_image(index)
+            self.process_db_data(img)
+
+        def on_evaluation_sample_changed(index):
+            print(f"on_evaluation_sample_changed={index}")
+            self.evaluation_sample_label.setText(f"Evaluation sample: {index}")
+            img = self.data_collector.get_evaluation_image(index)
+            self.process_db_data(img)
+
+        self.pause_button.clicked.connect(pause_callback)
+        self.image_view_type_combobox.currentIndexChanged.connect(on_image_view_type_change)
+        self.training_sample_slider.valueChanged.connect(on_training_sample_changed)
+        self.evaluation_sample_slider.valueChanged.connect(on_evaluation_sample_changed)
+
+        mode_options_layout = QVBoxLayout()
+        mode_options_layout.setAlignment(Qt.AlignTop)
+        mode_options_layout.addWidget(data_control_label)
+        mode_options_layout.addWidget(self.pause_button)
+        mode_options_layout.addWidget(image_view_label)
+        mode_options_layout.addWidget(self.image_view_type_combobox)
+
+        mode_options_layout.addWidget(self.training_sample_label)
+        mode_options_layout.addWidget(self.training_sample_slider)
+        mode_options_layout.addWidget(self.evaluation_sample_label)
+        mode_options_layout.addWidget(self.evaluation_sample_slider)
+
+        self.mode_options_group.setLayout(mode_options_layout)
+
+    def make_calibration_group(self):
+
+        self.calibration_group = QGroupBox("Calibration options")
+        self.calibration_group.setFlat(True)
+
+        self.calibrator_type_combobox = QComboBox()
+        self.calibrator_type_combobox.setEnabled(False)  # TODO(knzo25): implement this later
+
+        self.calibration_parameters_button = QPushButton("Calibration parameters")
+        self.calibration_button = QPushButton("Calibrate")
+        self.evaluation_button = QPushButton("Evaluate")
+        self.save_button = QPushButton("Save")
+        self.calibration_status_label = QLabel("Calibration status: idle")
```

Comment:
[nits] Is it better to separate these to another group from "Calibration options" group, like "Calibration status" ?

Reply:
Currently the group "calibration options" has the current elements:
 - Calibrator combobox
 - Calibration parameters button
 - Calibrate button
 - Evaluate button
 - Save button
 - Several calibration status-related labels.

Could you explain which ones you would like to separate? Only the labels?

As in:
Calibration options:
 - Calibrator combobox
 - Calibration parameters button
 - Calibrate button
 - Evaluate button
 - Save button

Calibration status:
 - Several calibration status-related labels.
 
 ?

Reply:
These are "Calibration status" ? by just my idea.
![image](https://user-images.githubusercontent.com/1531482/215025071-c451afe2-1320-4e2f-bc19-94dd76959bf3.png)

Reply:
Currently, "calibration status" is only the label from the first line of the image.

Is this what you mean?
![Screenshot from 2023-01-27 15-50-31](https://user-images.githubusercontent.com/4712270/215026454-97d1f306-35fa-4337-ad19-26b1e319789f.png)

Reply:
> Currently, "calibration status" is only the label from the first line of the image.

Hmm, that is correct. But aren't those "calibration options"? Is there a better name?

Reply:
If you refer to a more suitable group name for "Calibrate", "Evaluate", and "Save", how about Calibration control? actions?

Reply:
Sorry for your confusion. How about just changin group title simply?

- Calibration control
  - Camera model
  - Calibration parameters
  - Calibratie
  - Evaluate
  - Save
  - Calibration status
  - Calibration time
  - Training samples
  - Evaluation samples

Reply:
Addressed in [b202069](https://github.com/tier4/CalibrationTools/pull/65/commits/b20206942a125873bc65da08fc205a828a701a2f)

(Also fixed a bug)

Reply:
![image](https://user-images.githubusercontent.com/4712270/215384080-36c946b8-33c2-4afe-baa5-fb3084ac2cfa.png)

Reply:
Good!

Reply:
:tada: